examples/starboard/src/bot.c

```diff
@@ -0,0 +1,36 @@
+#include "bot.h"
+#include <concord/log.h>
+#include <stdlib.h>
+#include <string.h>
+#include "db.h"
+#include "commands.h"
+#include "starboard.h"
+
+static void star_bot_on_ready(struct discord* client, const struct discord_ready* event) {
+  char* discriminator = event->user->discriminator;
+  char* username = event->user->username;
+
+  if(strncmp(discriminator, "0", strlen(discriminator)) == 0) { // This is for the time, when Discord bots will lose their discriminators
```

Comment:
Perhaps remove and just add when they really do -- If they do?

Reply:
Also the syntax used in Concord has a space between `if` and `(`, might want to follow it here too. (as well as other things)

Reply:
Discord definetly will phase out tags for bots, but even if they don't, I think It's good future-proofing and can be easly modified once the transition happens.